src/main/java/com/puppycrawl/tools/checkstyle/gui/TreeTable.java

```diff
@@ -232,10 +230,10 @@ public void setEditor(JTextArea textArea) {
     /**
      * Sets line position map.
      * @param linePositionMap Line position map.
+     * @noinspection AssignmentToCollectionOrArrayFieldFromParameter
      */
     public void setLinePositionMap(List<Integer> linePositionMap) {
```

Comment:
argument type should ImmutableList, in other case it is not clear why we suppress violation